Plugins/Reqnroll.MSTest.Generator.ReqnrollPlugin/PackageReadme.md

```diff
@@ -0,0 +1,34 @@
+﻿Reqnroll.MSTest (pronounced as [reknroʊl]) enables using Reqnroll with MSTest v2 test framework.
```

Comment:
add a link to https://learn.microsoft.com/en-us/dotnet/core/testing/unit-testing-csharp-with-mstest